### ISSUE TITLE
Spelling mistakes in pb-intro.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6439]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6439|
|Type|defect|
|Created|2013-04-17 13:17:56Z|
|Branch|https://github.com/twisted/twisted/tree/pb-intro-type-6439|

`doc/core/howto/pb-intro.xhtml` says "posess" rather than "posses".

Attachments:

* [fixed_spelling_mistake.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/480/4802de61a70652cd389e566d0794ec61d34ee385/3213fb36a89175d63b98b8b0f20799eb82193584.patch) (629 bytes) - added by dhanush on 2013-04-17 13:30:35Z - Fixed the spelling mistake

<details><summary>Searchable metadata</summary>

```
trac-id__6439 6439
type__defect defect
reporter__itamar itamar
priority__lowest lowest
milestone__ 
branch__branches_pb_intro_type_6439 branches/pb-intro-type-6439
branch_author__dhanush dhanush
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy_gsoc easy gsoc
time__1366204676000000 1366204676000000
changetime__1366226724000000 1366226724000000
version__None None
owner__tom_prince tom.prince

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_2"></a>|@itamarst commented|
|-|-|

Hi Dhanush,

Given that you've already demonstrated you know the development process, could you leave these simple intro tickets for other candidates? Thanks!

**Comment by twisted-trac** [comment]
|<img alt="dhanush's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|dhanush commented|
|-|-|

Sure. I'll keep that in mind. :)

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_4"></a>|@tomprince commented|
|-|-|

(In [38123]) Branching to pb-intro-type-6439.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_5"></a>|@tomprince commented|
|-|-|

(In [38124]) Apply fixed_spelling_mistake.patch from dhanush.

Refs: [#6439](https://github.com/twisted/twisted/issues/6439)

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_6"></a>|@tomprince set owner to @tomprince|
|-|-|

This looks good. Thanks. Will merge.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_7"></a>|@tomprince set status to `closed`|
|-|-|

(In [38126]) Merge pb-intro-type-6439: Fix spelling mistakes in pb-intro.xhtml.

Author: dhanush
Reviewers: tom.prince
Fixes: [#6439](https://github.com/twisted/twisted/issues/6439)
